guide/src/installation.md

```diff
@@ -35,6 +88,15 @@ hermes version
 hermes 0.1.1
 ```
 
+
+__Troubleshooting__:
+In case the `cargo install` command above fails, as a first course of action we
+recommend trying to run the same command with the additional `locked` flag:
+
+```shell
+cargo install ibc-relayer-cli --bin hermes --locked
+```
+
```

Comment:
What do you think about always instructing users to supply the `--locked` flag? This ensures that the binary we run has the very same dependencies than the code that runs in CI as well as the released binaries.

Reply:
Yuss I like it.
b4090f0c3dd8ef862a1b33b54af7d3ed4b6715bf